test/system/post_test.rb

```diff
@@ -26,13 +26,29 @@ class PostTest < ApplicationSystemTestCase
       assert_selector('h1', text: "My new post")
       assert_selector('#content', text: "All about this interesting stuff")
       assert_selector('.alert-success', text: "×\nSuccess! Thank you for contributing open research, and thanks for your patience while your post is approved by community moderators and we'll email you when it is published. In the meantime, if you have more to contribute, feel free to do so.")
-      
+
     end
 
   end
 
+  test 'like button on the post' do
+    visit '/wiki/wiki-page-path/comments'
+
+    find('a[data-target="#loginModal"]', text: 'Login').click()
```

Comment:
It's a bit more readable here to just write:
`click_on "Login"`

Reply:
Not really, this will trigger the modal, `click_on "Login"` doesn't work.

Reply:
It works for me. Clicking the button automatically triggers the modal

Reply:
Let's not make Travis angry, it's readable enough :)